Lib/multiprocessing/connection.py

```diff
@@ -286,7 +301,13 @@ def _send_bytes(self, buf):
                 ov.cancel()
                 raise
             finally:
+                self._send_ov = None
                 nwritten, err = ov.GetOverlappedResult(True)
+            if err == WSA_OPERATION_ABORTED:
```

Comment:
What other value can it be? There is `assert err == 0` below, so I guess that any error was unexpected.

Could we simply check that `err` is not zero here?

Reply:
I chose to write a minimalist change: change at least code as possible. I introduce one new error, I added a check for this error, and that's all. I don't know the code enough to answer to your question. I'm not a multiprocessing or Windows API expert at all :-(